packages/bot-web-ui/src/components/draggable/draggable.tsx

```diff
@@ -44,11 +49,25 @@ export default function Draggable({
                 width,
                 height,
             }}
+            style={{
+                left: xaxis,
+                top: yaxis,
+            }}
             dragHandleClassName={dragHandleClassName}
             enableResizing={enableResizing}
             minHeight={height}
             minWidth={minWidth}
-            onDragStart={e => {
+            onDrag={(e, data) => {
+                //we need these calculations since we no longer use the 'transform: translate(x, y)' property
+                //as it causes unexpected behaviour of Beta Chart styles & helps avoid bounce bug upon the first drag
```

Comment:
here I left a comment that you guys know why we changed this component, it properly works for transaction details too